test/test_with_cplex.jl

```diff
@@ -163,6 +163,45 @@ end
     @test r["ElectricTariff"]["year_one_demand_cost_before_tax"] ≈ expected_demand_cost
 end
 
+@testset "ASHP" begin
+    d = JSON.parsefile("./scenarios/ashp.json")
```

Comment:
Is someone currently testing with CPLEX?!

Reply:
Deleted this. I performed this test for my own only but accidentally pushed it.

Reply:
yes, removed here - https://github.com/NREL/REopt.jl/pull/372/commits/05bcdc048fed964dab2e094c6144d087901e7abb